devtools/environment.yml

```diff
@@ -10,5 +10,7 @@ packages:
   - pandas
   - matplotlib-base
   - xlrd
+  - beautifulsoup4
```

Comment:
I couldn't use this file for creating the environment with the following command:
`conda env create -n kinodata -f environment.yaml`
Shouldn't `packages` be named `dependencies` and the pip stuff be on a different indentation level?